client-v2/src/actions/ArticleFragments.ts

```diff
@@ -262,28 +262,28 @@ const insertArticleFragmentWithCreate = (
 
 const removeArticleFragment = (
   type: string,
-  id: string,
+  collectionId: string,
   articleFragmentId: string,
   persistTo: 'collection' | 'clipboard'
 ): ThunkResult<void> => {
   return (dispatch: Dispatch, getState) => {
     const getGroupIdFromState = () => {
-      if (id === 'clipboard') {
-        return id;
+      if (collectionId === 'clipboard') {
```

Comment:
This ambiguity bothered me as I was writing, so I corrected it -- but it's not really relevant to this PR! I can split it out if necessary.

Reply:
I think it's useful to make the distinction 👍

Reply:
agree!